### DISCUSSION TITLE
Interpretation of DB Client Metrics

### DISCUSSION THREAD

**Discussion by FWinkler79** [open-telemetry/opentelemetry-specification#3039] [Q&A] [answered]
Hi there,

we looked at the [Database Metrics](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/database-metrics.md) and there is a disclaimer that states the following:

>The conventions described in this section are specific to SQL and NoSQL clients.
>Disclaimer: These are initial database client metric instruments and attributes but more may be added in the future. 

Now we understand these are client-related metrics, but what's not entirely clear to us is, who would measure / report them.
Are these metrics to be measured and reported by DB clients (looking at a server), or should DB servers (looking towards the client) report them?
Both are potentially possible, depending on the point of view.

What's the exact intention here?

Thanks!

**Comment by arminru** [comment, marked as answer]
For now, only metrics for monitoring connection pools maintained by DB clients are defined.
To my knowledge, I think only clients would be able to report them on their own, as DB servers will not have such insights on their clients, but there might be exceptions I'm not aware of.
